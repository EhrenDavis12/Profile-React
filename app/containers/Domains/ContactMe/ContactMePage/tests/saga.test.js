/**
 * Test sagas
 */

/* eslint-disable redux-saga/yield-effects */
// import { take, call, put, select } from 'redux-saga/effects';
// import contactMePageSaga from '../saga';

// const generator = contactMePageSaga();

describe('contactMePageSaga Saga', () => {
  it('Expect to have unit tests specified', () => {
    expect(true).toEqual(false);
  });
});
