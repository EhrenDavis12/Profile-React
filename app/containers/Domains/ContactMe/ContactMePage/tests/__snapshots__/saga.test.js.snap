// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navigationSaga Saga should execute the rootSaga 1`] = `
Object {
  "@@redux-saga/IO": true,
  "ALL": Array [
    GeneratorFunctionPrototype {
      "_invoke": [Function],
    },
  ],
}
`;

exports[`navigationSaga Saga should execute the submitMessageFormSaga 1`] = `
Object {
  "@@redux-saga/IO": true,
  "FORK": Object {
    "args": Array [
      "app/ContactMePage/SUBMIT_MESSAGE_FORM",
      [Function],
    ],
    "context": null,
    "fn": [Function],
  },
}
`;

exports[`navigationSaga Saga should execute the submitMessageFormStart 1`] = `
Object {
  "@@redux-saga/IO": true,
  "CALL": Object {
    "args": Array [
      "undefinedundefined",
      [Function],
      [Function],
      [Function],
    ],
    "context": null,
    "fn": [Function],
  },
}
`;
